airflow/api_connexion/schemas/task_instance_schema.py

```diff
@@ -171,7 +171,7 @@ class SetSingleTaskInstanceStateFormSchema(Schema):
     """Schema for handling the request of updating state of a single task instance."""
 
     dry_run = fields.Boolean(dump_default=True)
-    new_state = TaskInstanceStateField(required=True, validate=validate.OneOf([State.SUCCESS, State.FAILED]))
+    new_state = TaskInstanceStateField(required=True, validate=validate.OneOf([State.SUCCESS, State.FAILED, State.SKIPPED])
```

Comment:
You also need to update the state in the openapi specification https://github.com/apache/airflow/blob/985f3501806980e6612ae7fbe81669b001e93597/airflow/api_connexion/openapi/v1.yaml#L4244-L4249

Reply:
Ah good point, I saw that first before I got to the part I already edited above, but I guess I assumed this was automatic, done in 2 places, is that correct?